packages/build-info/src/bin.ts

```diff
@@ -1,31 +1,30 @@
-#!/usr/bin/env node
```

Comment:
We don't need the hashbang anymore?

Reply:
nope entry point is the `bin.js` in the package root